client/src/components/Search/index.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { AutoComplete, Select, Radio, Button } from 'antd';
+import world from 'full-countries-cities';
+import 'antd/dist/antd.css';
+
+import Keywords from '../Keywords';
+
+import './style.css';
+
+const { Option } = Select;
+
+class Search extends Component {
+  state = {
+    sitspot: '',
+    country: '',
+    city: '',
+    lookingFor: '',
+    keyword: '',
+    viewKeywords: false,
+  };
+
+  renderOptions = list => {
+    const countres = list.map(item => (
+      <Option key={item} value={item}>
+        {item}
+      </Option>
+    ));
+    return countres;
+  };
+
+  onSubmit = () => {};
+
+  setKeyword = value => {
+    if (value) this.setState({ keyword: value, viewKeywords: false });
+    else this.setState({ viewKeywords: false });
```

Comment:
unnecessary functionality since we separated the toggle function from the other function
```js
setKeyword = value => this.setState({ keyword: value, viewKeywords: false });
```
directly
and line 38 is redundant
https://github.com/GSG-G7/sitspot/blob/d6ba8234c3799a03a7f691cfc4492300423c0734/client/src/components/Search/index.js#L36